PyInstaller/utils/conftest.py

```diff
@@ -16,48 +16,40 @@
 # ---------------
 import copy
 import glob
+# Set a handler for the root-logger to inhibit 'basicConfig()' (called in
+# PyInstaller.log) is setting up a stream handler writing to stderr. This
+# avoids log messages to be written (and captured) twice: once on stderr and
+# once by pytests's caplog.
```

Comment:
This comment belongs with `logging.getLogger().addHandler(logging.NullHandler())` call, not the import.